static/styles/portico/footer_new_style.css

```diff
@@ -0,0 +1,146 @@
+footer {
+    background: #313257;
+    box-sizing: border-box;
+}
```

Comment:
We can probably call this `footer.css`. We should avoid having CSS for all `<footer>` elements though -- is this meant to be a `.footer`?

Reply:
Ok, used `.footer`.